src/Security/JsonWebTokenAuthenticator.php

```diff
@@ -63,7 +59,14 @@ public function start(Request $request, AuthenticationException $authException =
      */
     public function supports(Request $request)
     {
-        return $request->headers->has('X-JWT-Authorization');
+        if (!$request->headers->has('X-JWT-Authorization')) {
+            return false;
+        }
+
+        $authorizationHeader = $request->headers->get('X-JWT-Authorization');
+        $result = preg_match('/^Token (.*)$/', $authorizationHeader, $matches);
```

Comment:
your regexp could be more precise, this could eliminate the need for checking `$matches` and just rely on the return value of `preg_match`.

suggesting the following reg exp `/^Token \S+$/`.

Reply:
I looked at this for like 5 minutes and then threw in the towel, thanks for putting me on track here.